doc/release-notes/9356-rate-limiting.md

```diff
@@ -0,0 +1,12 @@
+Rate limiting has been added to prevent users from over taxing the system either deliberately or by runaway automated processes. Rate limiting can be configured on a tier level with tier 0 being reserved for guest users and tiers 1-any for authenticated users. Administrator/Superuser accounts are exempt from rate limiting.
+Rate limits can be imposed on command APIs by configuring the tier, the command, and the hourly limit in the database.
+Two database settings configure the rate limiting.
```

Comment:
Please add documentation for all new settings, including the ones described below, in installation/config.rst.

Reply:
Done